AjaxControlToolkit.SampleSite/Global.asax

```diff
@@ -11,6 +11,8 @@
             "~/Scripts/WebForms/MsAjax/MicrosoftAjaxTimer.js",
             "~/Scripts/WebForms/MsAjax/MicrosoftAjaxWebForms.js"));
 
+        // create "Temp" folder beforehand to avoid site recompilation: http://stackoverflow.com/questions/2699460/
```

Comment:
SampleSite is not set to run in Medium Trust, therefore setting `RootTempFolderPath` isn't necessary.
However, for documentation purposes, it could be a commented line with a corresponding note:

   > // In Medium Trust environments, set `AjaxFileUploadHelper.RootTempFolderPath` to an existing directory located within the web application root.

StackOverflow link is not necessary.